src/modules/accounts/views/ViewLogin.vue

```diff
@@ -114,14 +119,19 @@ async function handleLogin() {
           label="Email address"
           v-model="form.email"
           :error="emailError"
+          data-cy="your email address"
         />
-        <BaseInput
-          type="password"
-          name="password"
-          label="Password"
-          v-model="form.password"
-          :error="passwordError"
-        />
+        <div class="input_form">
+          <BaseInput
+            :type="showPassword ? 'text' : 'password'"
+            name="password"
+            label="Password"
+            v-model="form.password"
+            :error="passwordError"
+          />
+
+          <PasswordToggle @click="showPassword = !showPassword" />
```

Comment:
Replace the "@click" with the event emitted from the child component and change the state.